### ISSUE TITLE
My app's build fails if PRODUCT_NAME has a space using version 1.2.0.beta.1

### ISSUE THREAD

**Issue by mvalentiner** [CocoaPods/CocoaPods#6341]
> Before you start, are you using the latest CocoaPods release?
Yes, 1.1.1.
This is a bug with 1.2.0.beta.1

> A lot changes with Xcode releases that are not backwards compatible.
>
> Xcode 8 Status: Only works with CocoaPods 1.1.0+
I'm using 8.2.1.

* [ √] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

ℹ Please replace this with what you did.  
sudo gem install -n /usr/local/bin cocoapods --pre

ℹ Please replace this with what you expected to happen.  
My project should build without issue

ℹ Please replace this with of what happened instead.  
Build failed with this error:
```
Code Signing /Users/Mike/Library/Developer/Xcode/DerivedData/netresponse-gwkoqmabwrdvbedmmwrhjeeqatat/Build/Products/Debug-iphonesimulator/NRM Demo.app/Frameworks/AdobeMobileSDK.framework with Identity -

/usr/bin/codesign --force --sign -  --preserve-metadata=identifier,entitlements /Users/Mike/Library/Developer/Xcode/DerivedData/netresponse-gwkoqmabwrdvbedmmwrhjeeqatat/Build/Products/Debug-iphonesimulator/NRM Demo.app/Frameworks/AdobeMobileSDK.framework

/Users/Mike/Library/Developer/Xcode/DerivedData/netresponse-gwkoqmabwrdvbedmmwrhjeeqatat/Build/Products/Debug-iphonesimulator/NRM: No such file or directory
```
Note the last line: the name of the product that is trying to be signed is "NRM Demo", not "NRM".

I worked around the problem by uninstalling 1.2.0.beta.1 and reverting back to version 1.1.1.  My project builds as before.

ℹ Please replace this with the output of `pod env`.
Sorry, I uninstalled 1.2.0.beta.1, so I don't have the output of the environment that was causing the problem.
Here's the output of my current 1.1.1 working environment:
### Stack

```
   CocoaPods : 1.1.1
        Ruby : ruby 2.0.0p648 (2015-12-16 revision 53162) [universal.x86_64-darwin16]
    RubyGems : 2.6.7
        Host : Mac OS X 10.12.2 (16C67)
       Xcode : 8.2.1 (8C1002)
         Git : git version 2.10.1 (Apple Git-78)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib
Repositories : bitbucket-medtronic-medtronic-cocoapods - https://bitbucket.org/medtronic/medtronic-cocoapods.git @ b587c8d21e809fca492764bbae6a932443629033
               master - https://github.com/CocoaPods/Specs.git @ 6c7231f11940912847225cf72e5414fdf4ad2692
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.1
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.1.1
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '8.0'
use_frameworks!

abstract_target '' do
	pod 'AdobeMobileSDK', '4.13.2'
	pod 'Alamofire', '4.2.0'
	pod 'Crashlytics', '3.8.3'
	pod 'Fabric', '1.6.11'
	pod 'GCNetworkReachability', '1.3.2'
	pod 'MBProgressHUD', '1.0.0'
	pod 'SwiftyJSON', '3.1.3'
	pod 'XCGLogger', '4.0.0'
	pod 'Swinject', '2.0.0-beta.2'

	target 'NetResponseCommercial' do
	end

	target 'NetResponseVA' do
	end

	target 'NetResponseDevelop' do
	end

	target 'NRM Demo' do
	end

	target 'netresponseTests' do
	end
end
```

ℹ Please link to a project we can download that reproduces the issue.
Sorry, I don't have a link.  To recreate the problem, install 1.2.0.beta.1, go to your project file->Build Settings->Packaging->Product_Name and add a space to the name, do a pod install, then try and build.

**Comment by benasher44** [comment]
Hi @mvalentiner! Thanks for reporting this! This has already been fixed on master with #6160, and it will be available in an upcoming beta release. Have a great day!

**Comment by mvalentiner** [comment]
Cool!

--------------------------------------------
Michael Valentiner
michael.valentiner@gmail.com
